airflow/models/taskinstance.py

```diff
@@ -33,6 +33,7 @@
 import pendulum
 
 import dill
+from flask import url_for
```

Comment:
```suggestion
```

Don't need this anymore :)

Reply:
God damn it. Normally I would have `option + command + o`'d but since this would've changed so many lines I didn't do that.

Reply:
But now I did `optimize imports` :)